src/components/command-bar/commands/search/helpers/hit-counts-provider.tsx

```diff
@@ -7,7 +7,10 @@ import { ReactNode, createContext, useContext, useState } from 'react'
 
 const SearchHitsContext = createContext([])
 
-type HitCounts = Record<'docs' | 'tutorials' | 'integrations', number>
+type HitCounts = Record<
+	'docs' | 'tutorials' | 'integrations' | 'knowledgebase',
```

Comment:
What do you think about  using the existing `UnifiedSearchableContentType` to replace these unions (albeit with `Exclude` to yank 'global' out of there in this specific instance) instead?

```suggestion
	Exclude<UnifiedSearchableContentType, 'global'>
```